src/map/location.hpp

```diff
@@ -110,6 +112,18 @@ struct map_location {
 	map_location rotate_right_around_center(const map_location & center, int k) const;
 
 	friend std::size_t hash_value(map_location const &a);
+
+	int wml_x() const { return x + 1; }
+	int wml_y() const { return y + 1; }
+
+	void set_wml_x(int v) { x = v - 1; }
+	void set_wml_y(int v) { y = v - 1; }
```

Comment:
Well, I suppose removing `wml_x()` and `wml_y()` later, should they become unnecessary, would be less effort than it took to add them in the first place.